### PR TITLE
fix: getContextPath did not work correctly when called in a browser

### PR DESCRIPTION
**Pull request by DaveAtKeelvar**
### What change does this PR introduce?
The function `getContextPath` does not work correctly when called from inside a web browser. `process.env` is only available when the function is called within node.js. When called inside a web browser, we need to instead use `window._env_` to get access to the different context path variables.

### Why was this change needed?

To support custom context paths for Novu web

### CODE REVIEW

**Comment by DaveAtKeelvar** [comment]
Feel free to tidy up the TypeScript code, I haven't really used TypeScript before, and I'm sure it could be neater.

**Comment by vfxGer** [comment]
Good catch

File: libs/shared/src/config/contextPath.ts [hunk 1]
```diff
@@ -8,29 +8,41 @@ export enum NovuComponentEnum {
 export function getContextPath(component: NovuComponentEnum) {
   let contextPath = '';
 
-  if (process.env.GLOBAL_CONTEXT_PATH) {
-    contextPath += process.env.GLOBAL_CONTEXT_PATH + '/';
+  // Determine if we are running in the browser or in node.js. If we are
+  // running in node.js, we will have access to the process.env object,
+  // otherwise we will have access to the window._env_ object to get the
+  // environment variables.
```
Comment (scopsy) [inline libs/shared/src/config/contextPath.ts:14]: Seems like there is some eslint error here, i believe you would need to convert this to a multiline comment `/**`

Comment (DaveAtKeelvar) [reply libs/shared/src/config/contextPath.ts]: Thanks. I updated the code.

**Comment by DaveAtKeelvar** [comment]
@scopsy Is there any hope in moving this forward?

**Comment by scopsy** [comment]
@DaveAtKeelvar yes! Sorry about that, everytime I approve the  test runs and then while waiting forget to come backto it to merge. Added an alarm reminder 😅

**Pull request merged**